pkg/autoscaler/aggregation/bucketing_test.go

```diff
@@ -48,7 +48,7 @@ func TestTimedFloat64Buckets(t *testing.T) {
 			{trunc1.Add(3 * time.Second), pod, 1.0},        // nextnextnext bucket
 		},
 		want: map[time.Time]float64{
-			trunc1:                      1.0,
+			trunc1:                      2.0,
```

Comment:
I'd drop the `// same bucket` measures for the same pod here.

We can add a test that shows theoretical activator behavior where you have, say, a conc=1 poke for scale-from-0 and then a conc=10 stat for the "real deal".

Reply:
Done,.